shared/client/package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
```

Comment:
I would run `npm version patch` to bump the version for the `shared/client` package. You should probably increment the `dependency` entry in scheduler as well.

Before we get ready to release a new Scheduler version, we'll need to make sure the `shared/client` package gets updated on NPM.